image/image.go

```diff
@@ -38,6 +38,9 @@ import (
 	"github.com/snapcore/snapd/snap"
 	"github.com/snapcore/snapd/snap/squashfs"
 	"github.com/snapcore/snapd/store"
+
+	// important so that the testkeys get imported
```

Comment:
as I said we should organize things differently but we can work that out during RC, is not visible to users